### DISCUSSION TITLE
Richiesta di aiuto

### DISCUSSION THREAD

**Discussion by Lucaccc** [iisgiua/giuaschool#314] [Richieste di aiuto]
Salve, utilizzo Giuaschool da un anno nella mia scuola, ma con i nuovi aggiornamenti ho problemi con l'installazione e non so come fare.
Qualcuno potrebbe aiutarmi?

**Comment by manuelbarberi** [comment]
Guardi, posso aiutarla io... le spiego la procedura.... Deve cancellare tutto il contenuto dell'hosting e caricare i nuovi file .zip... Dopodichè bisogna eseguire la stessa procedura per installare il registro con la modifica che al posto di cliccare su installazione iniziale si clicca su aggiornamento... Dopo aver cliccato aggiornamento può seguire tranquillamente tutti i passaggi di configurazione.

**Reply by Lucaccc** [reply]
Salve, è ancora disponibile nell'aiutarmi a svolgere la procedura di installazione? 
La ringrazio.

**Reply by manuelbarberi** [reply]
Si, mi mandi tutte le info sulla mail: giuaschoolregistro@gmail.com

**Reply by Lucaccc** [reply]
Fatto.
Le ho inviato tutto via mail. Per qualunque cosa sono a disposizione.

**Reply by Lucaccc** [reply]
Ha ricevuto la mail?
@manuelbarberi
